### PR TITLE
Added Noto Sans Sanskrit font to package.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Adds Noto Sans font to codebase

# Needed By (Date)
- Version 5

# Urgency
- low

# Steps to Test

1. Check out this branch
2. Compile styleguide
3. Review typography section in rendered styleguide

# Associated Issues and/or People
- See #274 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: core/scss/core/fonts/_fonts.scss [hunk 1]
```diff
@@ -19,6 +19,9 @@
 // Script font: Kalam.
 @import url('https://fonts.googleapis.com/css?family=Kalam:300,400,700');
 
+// Sandscrit font: Noto Sans
+@import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,400italic,700,700italic');
```
Comment (yvonnetangsu) [inline core/scss/core/fonts/_fonts.scss]:
```suggestion
@import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,400i,700,700i&subset=devanagari');
```

Comment (yvonnetangsu) [reply core/scss/core/fonts/_fonts.scss]: It seems like by default it only loads the latin subset so if we need sanskrit support we might need to add subset=devanagari at the end.

File: core/scss/core/fonts/_fonts.scss [hunk 2]
```diff
@@ -19,6 +19,9 @@
 // Script font: Kalam.
 @import url('https://fonts.googleapis.com/css?family=Kalam:300,400,700');
 
+// Sandscrit font: Noto Sans
```
Comment (yvonnetangsu) [inline core/scss/core/fonts/_fonts.scss]:
```suggestion
// Sanskrit font: Noto Sans
```

Comment (yvonnetangsu) [inline core/scss/core/fonts/_fonts.scss]:
```suggestion
// Sanskrit font: Noto Sans
```

File: core/scss/utilities/variables/core/_typography.scss [hunk 3]
```diff
@@ -206,6 +206,17 @@ $font-slab: 'Roboto Slab', $font-stack-georgia !default;
 //
 $font-handwriting: 'Kalam', $font-stack-helvetica !default;
 
+//
+// $font-sandscrit
```
Comment (yvonnetangsu) [inline core/scss/utilities/variables/core/_typography.scss]:
```suggestion
// $font-sanskrit
```

File: core/scss/utilities/variables/core/_typography.scss [hunk 4]
```diff
@@ -206,6 +206,17 @@ $font-slab: 'Roboto Slab', $font-stack-georgia !default;
...
+//
+// font-script Font family.
+//
+// Markup: $font-sandscrit: 'Noto Sans', $font-stack-helvetica !default;
```
Comment (yvonnetangsu) [inline core/scss/utilities/variables/core/_typography.scss]:
```suggestion
// Markup: $font-sanskrit: 'Noto Sans', $font-stack-helvetica !default;
```

File: core/scss/utilities/variables/core/_typography.scss [hunk 5]
```diff
@@ -206,6 +206,17 @@ $font-slab: 'Roboto Slab', $font-stack-georgia !default;
...
+//
+// Style guide: Variables.Core.Typography.font-sandscrit
+//
+$font-sandscrit: 'Noto Sans', $font-stack-helvetica !default;
```
Comment (yvonnetangsu) [inline core/scss/utilities/variables/core/_typography.scss]:
```suggestion
$font-sanskrit: 'Noto Sans', $font-stack-helvetica !default;
```

**Comment by josephgknox** [comment]
Pushed up a commit adding a mixin and class for the new font (mirroring what we've done for the other fonts).

**Comment by sherakama** [comment]
@yvonnetangsu Yeah, I am not sure where that duplication of the fonts section started but it has now been consolidated into one. An example has been added to the twig file.

**Comment by josephgknox** [comment]
@sherakama thanks! This looks good to me now (cc: @yvonnetangsu).

**Comment by yvonnetangsu** [comment]
Looks great! Good to go 😺

**Pull request merged**